specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-03-01-preview/examples/ApiManagementCreateODataApi.json

```diff
@@ -0,0 +1,77 @@
+{
+  "parameters": {
+    "serviceName": "apimService1",
+    "resourceGroupName": "rg1",
+    "api-version": "2023-03-01-preview",
+    "subscriptionId": "subid",
+    "apiId": "tempgroup",
+    "parameters": {
+      "properties": {
+        "description": "apidescription5200",
+        "displayName": "apiname1463",
+        "type": "odata",
+        "serviceUrl": "https://services.odata.org/TripPinWebApiService",
+        "path": "odata-api",
+        "protocols": [
+          "http",
+          "https"
+        ],
+        "format": "odata-link",
+        "value": "https://services.odata.org/TripPinWebApiService/$metadata"
+      }
+    }
+  },
+  "responses": {
+    "201": {
+      "body": {
+        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.ApiManagement/service/apimService1/apis/apiid9419",
+        "type": "Microsoft.ApiManagement/service/apis",
+        "name": "apiid9419",
+        "properties": {
+          "displayName": "apiname1463",
+          "apiRevision": "1",
+          "description": "apidescription5200",
+          "serviceUrl": "https://services.odata.org/TripPinWebApiService",
+          "type": "odata",
+          "path": "odata-api",
+          "protocols": [
+            "https",
+            "https"
+          ],
+          "authenticationSettings": null,
+          "subscriptionKeyParameterNames": null,
+          "isCurrent": true,
+          "isOnline": true
+        }
+      }
+    },
+    "202": {
+      "headers": {
+        "location": "https://management.azure.com/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.ApiManagement/service/apimService1/apis/tempgroup?api-version=2023-03-01-preview&asyncId=5c730e343244df1b9cb56e85&asyncCode=201"
+      }
+    },
+    "200": {
+      "body": {
+        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.ApiManagement/service/apimService1/apis/apiid9419",
+        "type": "Microsoft.ApiManagement/service/apis",
+        "name": "apiid9419",
+        "properties": {
+          "displayName": "apiname1463",
+          "apiRevision": "1",
+          "description": "apidescription5200",
+          "serviceUrl": "https://services.odata.org/TripPinWebApiService",
+          "type": "odata",
+          "path": "odata-api",
+          "protocols": [
+            "http",
+            "https"
+          ],
+          "authenticationSettings": null,
+          "subscriptionKeyParameterNames": null,
```

Comment:
curious questions... would you actually return null in your response, or empty settings, or just omit them?

Reply:
actually null